Owncloud iOs Client/Tabs/SettingTab/Account/Cell/AccountCell.m

```diff
@@ -40,16 +41,13 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated
     // Configure the view for the selected state
 }
 
+/*
```

Comment:
Commented Code

Reply:
👍 Removed, and also removed remaining references in xib file, account cell classes an setting class